src/containers/BooksForm.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createBook } from '../actions/index';
+import './styles/BooksForm.css';
+
+const bookCategories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+class BooksForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      category: bookCategories[0],
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ [event.target.id]: event.target.value });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const newBook = this.state;
+    const { createBook } = this.props;
+    createBook(newBook);
```

Comment:
As createBook mutates its parameter we are mutating state here. We should never mutate state because it leads to unexpected results. Instead we should pass a copy of the state to createBook or make createBook a pure function.